src/components/DatePicker/DatePicker.visual.tsx

```diff
@@ -7,48 +7,67 @@ visualize('DatePicker', () => {
   story('simple', () => {
     snap(
       'default props',
-      <DatePicker value={new Date('2019/12/15')} onChange={() => {}} />,
+      <div style={{ width: '280px' }}>
+        <DatePicker value={new Date('2019/12/15')} onChange={() => {}} />
+      </div>,
+    );
+    snap(
+      'responsiveness',
+      <div style={{ width: '600px' }}>
+        <DatePicker value={new Date('2019/12/15')} onChange={() => {}} />
+      </div>,
     );
     snap(
       'FilterDate - Prior for today dates (includes today)',
-      <DatePicker
-        value={new Date('2019/12/15')}
-        filterDate={date => date <= new Date('2019/12/15')}
-        onChange={() => {}}
-      />,
+      <div style={{ width: '280px' }}>
+        <DatePicker
+          value={new Date('2019/12/15')}
+          filterDate={(date) => date <= new Date('2019/12/15')}
+          onChange={() => {}}
+        />
+      </div>,
     );
     snap(
       'FilterDate - future dates only (includes today)',
-      <DatePicker
-        value={new Date('2019/12/15')}
-        filterDate={date => date >= new Date('2019/12/15')}
-        onChange={() => {}}
-      />,
+      <div style={{ width: '280px' }}>
+        <DatePicker
+          value={new Date('2019/12/15')}
+          filterDate={(date) => date >= new Date('2019/12/15')}
+          onChange={() => {}}
+        />
+      </div>,
     );
     snap(
       'Months dropdown',
-      <DatePicker
-        value={new Date('2019/12/15')}
-        showMonthDropdown
-        onChange={() => {}}
-      />,
+      <div style={{ width: '280px' }}>
+        <DatePicker
+          value={new Date('2019/12/15')}
+          showMonthDropdown
+          onChange={() => {}}
+        />
+      </div>,
     );
     snap(
       'Years dropdown',
-      <DatePicker
-        value={new Date('2019/12/15')}
-        showYearDropdown
-        onChange={() => {}}
-      />,
+      <div style={{ width: '280px' }}>
+        <DatePicker
+          value={new Date('2019/12/15')}
+          showYearDropdown
+          onChange={() => {}}
+        />
+        ,
+      </div>,
     );
     snap(
       'Months and years dropdown',
-      <DatePicker
-        value={new Date('2019/12/15')}
-        showMonthDropdown
-        showYearDropdown
-        onChange={() => {}}
-      />,
+      <div style={{ width: '280px' }}>
+        <DatePicker
+          value={new Date('2019/12/15')}
+          showMonthDropdown
+          showYearDropdown
+          onChange={() => {}}
+        />
+      </div>,
```

Comment:
Not critical at all, but maybe it could be easier to extract the component with the wrapping div.
Meaning:
```jsx
const VisualTestDatePicker = (props) => {
  const { width, ...rest } = props;

  return (
    <div style={{ width }} >
      <DatePicker {...rest} />
    </div> 
  );
};
...
snap('foobar', <VisualTestDatePicker ... >);
```

Again it's not critical for now